pillar/fluentd/consul.sls

```diff
@@ -15,24 +14,8 @@ fluentd:
             - pos_file: /var/log/syslog.pos
             - format: syslog
             - tag: consul.server
-        - directive: filter
-          directive_arg: '**'
-          attrs:
-            - '@type': grep
-            - regexp1: ident consul
         - {{ auth_log_source('syslog.auth', '/var/log/auth.log') }}
-        - {{ auth_log_filter('grep', 'ident', 'CRON') }}
+        - {{ auth_log_filter('grep', 'ident', '/consul', 'consul.server') }}
```

Comment:
I don't think we want to use this block for consul because the directive that you're removing to replace this with was to set it to only pass through consul logs, whereas this will pass through everything _except_ consul logs.